tests/test_tasks.py

```diff
@@ -239,6 +239,19 @@ def test_filter_manifest_pythons():
     assert len(manifest) == 1
 
 
+def test_filter_manifest_pythons_not_found(caplog):
+    config = _options.options.namespace(
+        sessions=(), pythons=("1.2",), keywords=(), posargs=[]
+    )
+    manifest = Manifest(
+        {"foo": session_func_with_python, "bar": session_func, "baz": session_func},
+        config,
+    )
+    return_value = tasks.filter_manifest(manifest, config)
+    assert return_value == 3
+    assert "Python version selection caused no sessions to selected." in caplog.text
```

Comment:
```suggestion
    assert "Python version selection caused no sessions to be selected." in caplog.text
```

Looks like this is just a typo causing a failure :)